pkg/pod/pod.go

```diff
@@ -133,9 +133,9 @@ func (b *Builder) Build(ctx context.Context, taskRun *v1beta1.TaskRun, taskSpec
 	// Convert any steps with Script to command+args.
 	// If any are found, append an init container to initialize scripts.
 	if alphaAPIEnabled {
-		scriptsInit, stepContainers, sidecarContainers = convertScripts(b.Images.ShellImage, steps, taskSpec.Sidecars, taskRun.Spec.Debug)
+		scriptsInit, stepContainers, sidecarContainers = convertScripts(b.Images.ShellImage, b.Images.ShellImageWin, steps, taskSpec.Sidecars, taskRun.Spec.Debug)
```

Comment:
Should windows support go into the alpha feature gate, alongside debug support? 🤔

Reply:
Good point.
I changed this call because I changed the signature of the convertScripts method, but we could pass nil and check for that before we do any windows checks?

Reply:
Passing nil or an empty string for the windows shell image when `alphaAPIEnabled` is `false` would work ok I think. An alternative would be to write an entirely windows-focused `convertScripts` implementation and only invoke it if `alphaAPIEnabled` is `true` and `requiresWindows` would be true. Might involve a bit more duplication or end up being more lines but it might reduce the number of windows-specific branches? Could also push a change like that back to a refactor pr in future as well though.